pages/faq/troubleshooting/faq.md

```diff
@@ -63,24 +68,19 @@ Starting from {{ $names.os.lower }} v2.0.7, the devices connect to the following
 - 3.resinio.pool.ntp.org
 
 ##### What network ports are required?
-
 In order for a {{ $names.company.lower }} device to get outside of the local network and connect to the {{ $names.company.lower }} API, there are a few core network requirements.
 
 {{ $names.company.upper }} makes use of the following ports:
 
-* `443` TCP - This is the most fundamental requirement - it is used to connect to the VPN and the web terminal, and many web endpoints using TLS (https://.)
+* `443` TCP - This is the most fundamental requirement - it is used to connect to the VPN and the web terminal, and many web endpoints using TLS (`https://`.)
 * `123` UDP - For NTP time synchronisation.
 * `53` UDP - For DNS name resolution.
 
 Each of these should work with outward only (and inward once outward connection established) firewall settings.
 
-Additionally, you should whitelist the following domains for the relevant ports above:
-* `*.{{ $names.dashboard_domain }}`
-* `*.docker.com`
```

Comment:
Why did we remove docker.com domains? these are needed currently for host OS upgrades

Reply:
Or this...